helix-core/src/main/java/org/apache/helix/task/AbstractTaskDispatcher.java

```diff
@@ -74,6 +75,12 @@ public void updatePreviousAssignedTasksStatus(
       Set<Integer> skippedPartitions, WorkflowControllerDataProvider cache,
       Map<String, Set<Integer>> tasksToDrop) {
 
+    // If a job is in the following states and contains running tasks, all of the running tasks
+    // should go to the ABORTED states.
```

Comment:
Nit: 

If a job is in one of the following states and its tasks are in RUNNING states, the tasks will be aborted.

Reply:
Fixed. Thanks